src/samplers/ebpf/tcp/statistics.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+pub enum Statistic {
+    ConnectLatency,
+}
+
+impl std::fmt::Display for Statistic {
```

Comment:
Seems a bit strange to use Display for the metric name, which it looks like is what's happening here? Why's it set up like this?

Reply:
The metrics registry is keyed on strings. By implementing `Display` we allow for conversion with `to_string()`. In the future, we should probably have our own trait for statistics. Open to ideas on how to improve this.

Reply:
Yeah, moving this to a different trait sounds better. Will keep thinking a bit about it as I look more, I don't have a ton of ideas off the top of my head.